master/custom/workers.py

```diff
@@ -135,9 +155,9 @@ def get_workers(settings):
             parallel_tests=40,
         ),
         cpw(
-            name="cstratak-RHEL8-fips-x86_64",
```

Comment:
Did you remove this config on purpose?

Reply:
Moved it up together with the other x86_64 ones.